build.gradle

```diff
@@ -0,0 +1,39 @@
+plugins {
+    id 'java'
+    id "application"
+}
+
+repositories {
+    // First look for local Apache James dependencies. We can publish code changes locally and work with them
+    // See caveats https://docs.gradle.org/current/userguide/declaring_repositories.html#sec:case-for-maven-local
+    // Maybe avoid mavenLocal and use a directory with jars.
+    // mavenLocal()
+
+    // Next look into snapshots and staging
+    // https://infra.apache.org/repository-faq.html
+
+    maven { url "https://repository.apache.org/content/groups/public/" }
+    maven { url "https://repository.apache.org/content/groups/staging/" }
+    maven { url "https://repository.apache.org/content/groups/snapshots/" }
+
+    mavenCentral()
+    jcenter()
+}
+
+ext {
+    jamesVersion = '3.5.0-SNAPSHOT'
+}
+
+dependencies {
+
+    compile "org.apache.james:james-server-jpa-guice:${jamesVersion}"
+    compile "org.apache.james:james-server-cli:${jamesVersion}"
+
+}
```

Comment:
Is this intended to compile James? Does this mean that the gradle build file will be placed in the James project? Is that the idea?

Reply:
No, those dependencies. I think I should make them "implementation" and it should work the same. 

I'm abusing the syntax a bit. I need to tell gradle what my dependencies are.